frontend/src/services/vdyp-api/apis/run-hcsvprojection-api.ts

```diff
@@ -0,0 +1,170 @@
+/* tslint:disable */
+/* eslint-disable */
+import globalAxios from 'axios'
+import type { AxiosResponse, AxiosInstance, AxiosRequestConfig } from 'axios'
+import { Configuration } from '../configuration'
+// Some imports not used depending on template conditions
+// @ts-ignore
+import { BASE_PATH, BaseAPI, RequiredError } from '../base'
+import type { RequestArgs } from '../base'
+import type { FileUpload, Parameters } from '../models'
+
+export const RunHCSVProjectionApiAxiosParamCreator = function (
```

Comment:
Same as previous comment, should this be a promise? Doesn't seem to execute anything in an async way